spec/models/child_spec.rb

```diff
@@ -56,6 +56,7 @@
 
   context 'scopes' do
     let(:inactive_child) { create(:child, active: false) }
+    let(:deleted_child) { create(:child, deleted: true) }
 
     it 'only displays active children in the active scope' do
       expect(Child.active).to include(child)
```

Comment:
```suggestion
      expect(Child.active).to include(child)
      expect(Child.active).to not_include(deleted_child)
      expect(Child.active).to not_include(inactive_child)
```

Reply:
I think deleted entries should be included in any scope including `active`, unless the `not_deleted` scope is explicit. One alternative would be to adjust `active` scope to be `where(active: true, deleted: false)` , which seems not great, or to more generally apply `not_deleted` as a default scope everywhere and provide an inversion `include_deleted`. I think the hassle of adding `not_deleted` everywhere is worth it for avoiding the possible confusion. Also it looks like default scopes are frowned upon: https://piechowski.io/post/why-is-default-scope-bad-rails/